Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -2802,13 +2802,15 @@ Ref<Document> AXObjectCache::protectedDocument() const
 VisiblePosition AXObjectCache::visiblePositionForTextMarkerData(const TextMarkerData& textMarkerData)
 {
     RefPtr node = nodeForID(textMarkerData.axObjectID());
-    if (!node)
-        return { };
-
-    if (node->isPseudoElement())
+    if (!node || node->isPseudoElement())
         return { };
 
-    auto visiblePosition = VisiblePosition({ node.get(), textMarkerData.offset, textMarkerData.anchorType }, textMarkerData.affinity);
```

Comment:
Isn't this:

`{ node.get(), textMarkerData.offset, textMarkerData.anchorType }`

Calling the `Position` constructor with whatever anchor type is in the text marker? That seems contrary to what you wrote in the commit message, which makes me confused:

"AXObjectCache::visiblePositionForTextMarkerData was creating a Position with PositionIsOffsetInAnchor regardless the anchorType passed in the TextMarkerData"